nlpserver.py

```diff
@@ -12,7 +12,7 @@
 #  configurations
 #app.config['var1'] = 'test'
 app.config["YOLO_MODEL_NAME"] = "yolov8m.pt"
-app.config["MOBILENET_MODEL_NAME"] = "mobilenet_v3_small.tflite"
+app.config["MOBILENET_MODEL_NAME"] = "mobilenet_v3_large.tflite"
```

Comment:
👍

Reply:
Really a matter of space/taste/ The small one really had little inference capabilities. But I will defer to you on this